### PR TITLE
Release 1.2.11

### PR DESCRIPTION
**Pull request by rodinux**
## Problem

- *Release from the stable branch of paheko, since the testing branch is developed  with 1.3.0*

## Solution

- *add the the new tarball, also the v2 compatibility for yunohost*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by rodinux** [comment]
!testme

**Comment by rodinux** [comment]
Hello @alexAubin @ericgaspar @fflorent , could someone approve this PR ? thanks

**Comment by rodinux** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -183,38 +97,12 @@ ynh_script_progression --message="Finalise upgrade" --weight=1
...
-	chmod +x /tmp/$script_post_migration
-	(cd /tmp; echo "/tmp/$script_post_migration > /tmp/$script_post_migration.log 2>&1" | at now + 2 minutes)
-fi
+sleep 5
 
 #=================================================
 # END OF SCRIPT
 #=================================================
+# remove directory duplicated
+ynh_secure_remove --file="$install_dir/data/data"
```
Comment (alexAubin) [inline scripts/upgrade:106]: Hmmm that one is a bit strange but okay I guess ?

**Review by alexAubin — APPROVED** [review]

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    help.en = "The SSO is not implemented for this app and YunoHost is not sure for hundred of users..."
+    help.fr = "Le SSO n'est pas implémenté pour cette application et YunoHost n'est pas sûr pour une centaine d'utilisateurs..."
+    type = "group"
+    default = "visitors"
+
+[resources]
+        [resources.sources.main]
+        url = "https://fossil.kd2.org/paheko/uv/paheko-1.2.11.tar.gz"
+        sha256 = "9d8a505d9165c38ee0d262316ba636536ebddf0a6127b2e731bc231fe053f28d"
+        autoupdate.strategy = "latest_github_tag"
```
Comment (Tagadda) [inline manifest.toml:47]: This will not work since the repo is not hosted on github.
```suggestion
```

**Review by Tagadda — CHANGES_REQUESTED** [review]

File: manifest.toml (hunk 2 above)
Comment (rodinux) [reply manifest.toml:47]: You are right, the mirror here https://github.com/YunoHost-Apps/nextcloud_ynh/issues/602 seems no more maintained...

**Review by Tagadda — APPROVED** [review]

**Pull request merged**